### PR TITLE
fix: update readme file to show all commands

### LINKED ISSUE #143: Add "How to start this project" section in  readme
Improve readme by including npm/yarn... commands with descriptions.

This is  NextJS app and it uses default commands

### PR DESCRIPTION
**Pull request by Potato-29**
# Pull Request Type
[ ] New feature
[x] Bug fix

This PR closes #143 

# 👨‍💻 Describe your changes
1. Rewrote the README and display all necessary commands

# ✔️ Check List (Check all the applicable boxes)

- [x] I tested my code
- [x] The title and description of the PR is clear and explains what has been done.

# Additional notes

# 📷 Screenshots

## Summary by Sourcery

Fix the README file to display all necessary commands and provide a detailed overview of the project setup and usage.

Bug Fixes:
- Update the README file to include all necessary commands for setting up, running, building, and linting the project.

Documentation:
- Rewrite the README to provide a comprehensive overview of the project, including prerequisites and available scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+```
+
+Runs the production build. You must run npm run build before using this command.
+
+##
+
+To run the linter, run:
+```bash
+npm run lint
+````
```
[the remark this answers is not in the document]
Comment (Potato-29) [reply README.md:45]: will fix

File: README.md [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
+Runs the linter to identify and fix code formatting issues.
 
-https://notion-with-widgets.vercel.app/
```
Comment (mslepko) [inline README.md:11]: @Potato-29 can you please keep the URL in the new readme?

**Review by mslepko — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Potato-29) [reply README.md:11]: sure, I'll update that and push it.

**Review by mslepko — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Potato-29) [reply README.md:11]: I've added the Live URL in the Overview section of readme

**Review by mslepko — APPROVED** [review]

**Comment by mslepko** [comment]
Thank you @Potato-29

**Pull request merged**